### ISSUE TITLE
Fish population abundance and oceanic variation

### ISSUE THREAD

**Issue by aebudden** [NCEAS/oss-2017#19]
Author: **Kathryn Ireland**
Topics: Fish populations

## Summary
Through discussions with LGL Ecological Research Associates, I have become interested in how synthesis research could contribute to sustainable fisheries management in the Gulf of Mexico. In 2014, researchers at LGL published a study examining the links between shrimping fishing effort and trends in fishery-independent data on trends in population abundances of several fish species. Their results suggested that Atlantic croaker showed a general increase in population abundance following a reduction in shrimping effort in 2003.  Despite the apparent link between Atlantic croaker populations and shrimping effort, an analysis of life history traits suggested that Atlantic croaker populations should be resilient to the impacts of the fishery due to the species’ high intrinsic growth rate. Several other species of fish comprising at least 5% of the shrimp fisheries’ bycatch showed little relationship between population fluctuations and shrimp fishing effort.  However, the authors of this study noted several statistical issues with this initial study, including the need to control for confounding environmental influences on fish population and to account for spatial and temporal autocorrelation.

Before we can understand the impact of any disturbance on the fisheries in the Gulf of Mexico, such as fishing or an event like the Deepwater Horizon Oil Spill, we need to better understand the historic range of variation and environmental influences on fish populations. The Southeast Area Monitoring and Assessment Program (SEAMAP) data collected by the National Marine Fisheries Service provides a wealth of information that could be used to better understand the variability and environmental influences on the variability on fish populations within the Gulf of Mexico. 

## Data Acquisition and Methods
I propose using the publically available Fall Shrimp/Groundfish trawl surveys to improve our understanding of the environmental factors influencing fish population in the Gulf of Mexico. To extend the previous work done by LGL, I propose building general linear regression models (glms) linking fish population abundances to oceanographic variables, such as the loop current, salinity, or sea surface temperatures. Because the data in the SEAMAP dataset are often correlated with previous values, I propose removing temporal autocorrelation (for example, using ARIMA modeling methods) to reduce the likelihood of overestimating significance between fish population abundances and oceanographic explanatory variables.

## Impact and Significance
The models linking fish population abundance to oceanographic information could provide a better understanding of the historic range of variability of these populations. Once baseline models have been developed, and if time allows, fishing effort or time since the Deepwater Horizon Oil Spill could be incorporated to get a better understanding of how the fisheries are responding to human disturbances. Sustainable fishery management can be impeded by uncertainty in fish population dynamics and the impact of management practices. Using the long-term datasets available within the Gulf of Mexico provides an opportunity to improve our understanding of the population dynamics of fish species to improve sustainability of fisheries in the region.

**Comment by adellapa** [comment]
I really like this approach. How about including also data on Chlorophyll and dissolved oxygen, if available from other database?

**Comment by demutsertlab** [comment]
I agree with previous comment that that are good additions. I know dissolved oxygen is collected during SEAMAP cruises as well, leaving Chl a as something that needs to come from another source.

**Comment by adellapa** [comment]
Data on Chl a can be downloaded from the NASA Earth Observations website at https://neo.sci.gsfc.nasa.gov/view.php?datasetId=MY1DMM_CHLORA

**Comment by ailich** [comment]
I'd be very interested in working on this project. It's very important to have this type of baseline understanding on what controls fish distributions at broad scales. The objectives are very clearly stated and from the proposal and comments it seems like the data needed is readily available.
